marquez_airflow/dag.py

```diff
@@ -174,7 +177,11 @@ def report_task(self,
         # If no extractor found or failed to extract metadata,
         # report the task metadata
         if not steps_metadata:
-            steps_metadata = [StepMetadata(task_name)]
+            steps_metadata = [StepMetadata(
+                name=task_name,
+                context={
+                    'task_info': inspect.getmembers(task)
```

Comment:
for those airflow specific context attributes, I would prefix with `airflow.`
for example here: `airflow.task_info`
how about adding `airflow.operator` as well?

Reply:
Sure, I like the `airflow.*` idea 👍